src/Indicator.vala

```diff
@@ -200,9 +213,16 @@ public class Power.Indicator : Wingpanel.Indicator {
 
     private bool show_notification () {
         if (is_in_session) {
-            var notification = new Notify.Notification ("indicator-power", "", "display-brightness-symbolic");
-            notification.set_hint ("x-canonical-private-synchronous", new Variant.string ("indicator-power"));
-            notification.set_hint ("value", new Variant.int32 (dm.brightness));
+            var notification = new Notify.Notification ("", "", "");
```

Comment:
```suggestion
            Notify.Notification notification;
```

Substitution is done in the following lines so we just need to prepare the variable here